R/get_aux_table.R

```diff
@@ -2,11 +2,20 @@
 #'
 #' @param data_dir character: Data directory
 #' @param table character: Name of auxiliary table
+#' @param long_format logical: do you want data long format ? (default FALSE)
 #'
 #' @return data.frame
 #' @export
 #'
-get_aux_table <- function(data_dir, table) {
+get_aux_table <- function(data_dir, table, long_format = FALSE) {
+  if (long_format && !table %in% c('cpi', 'ppp', 'gdp', 'pce', 'pop')) {
+    # stop('Please select one of cpi, ppp, gdp, pce and pop tables to get data in long format.')
+    # This gives a 500 error when using the API. This should be caught at the
+    # API filter level and give an informative message to the user.
+    # Forcing long_format to FALSE instead as a temporary work-around
+    # See https://github.com/PIP-Technical-Team/pipapi/issues/290
+    long_format <- FALSE
+  }
```

Comment:
Forcing to FALSE instead when a non-supported table is selected so it does not trigger a 500 error. See issue #290 for more info.